src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,118 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int START_ARRAY_SIZE = 10;
+    private int size = 0;
+    private int index;
```

Comment:
we don't need this field

Reply:
if you mean field index, it's important, because we use it in this separate method   "" private void checkIndexInBounds() ""
otherwise it's impossible to use it in method

Reply:
If you need to pass it as a parameter checkIndexInBounds(int index)